assets/js/page-enhancements.js

```diff
@@ -110,3 +85,58 @@
     $('#print-coversheets-content #downloadPdf').each((idx, element) => element.click())
   })
 })($, document)
+;(autoFocusBarcodeField = ($, document) => {
+  $(document).ready(() => {
+    $('#scan-barcode-form #barcode').focus()
+  })
+})($, document)
+;(autoFocusPrisonNumberField = ($, document) => {
+  $(document).ready(() => {
+    $('#find-recipient-by-prison-number-form #prisonNumber').focus()
+  })
+})($, document)
+;(autoFocusPrisonerNameField = ($, document) => {
+  $(document).ready(() => {
+    $('#find-recipient-by-prisoner-name-form #prisonerName').focus()
+  })
+})($, document)
+;(autoFocusContactPrisonerNameField = ($, document) => {
```

Comment:
God, this autofocus stuff is getting out of hand. We cant keep doing this for every page - I bet the other apps dont do this?
I wonder if we should challenge the requirement to autofocus fields. Notwithstanding the possible accessibility issues and already proven UX issues, this just feels like the wrong thing to be doing

Reply:
We could replace this with some JS that works for any screen - either focus on the first input field or the first input field in error - at least then we have consistency throughout the app. But I agree, it doesn't appear to have been thought out properly.